README.md

```diff
@@ -83,25 +79,64 @@ public string UserName()
 
 If you want to add roles to the `User` property you can have a look in `Transformers/ClaimsTransformer.cs` in the Sample project. There you can see an example how to get started with this.
 
-### Custom options
+### Configure options via configuration (recommended)
+
+You can use the default behavior of asp.net core to configure EasyAuth. You must only change in your `Startup.cs` the `.AddEasyAuth()` to `.AddEasyAuth(this.Configuration)`.
+
+> To get the property `this.Configuration` in your `Startup.cs` you must add `IConfiguration configuration` to your constructor parameters and create a property.
+
+To configure you providers you simple add the following to your appsettings.json. (or to your environment variables, or other [configuration sources](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/).)
+
+```json
+"easyAuthOptions": {
+    "AuthEndpoint": ".auth/me",
+    "providerOptions": [
+      {
+        "ProviderName": "EasyAuthForAuthorizationTokenService",
+        "Enabled": true,
+        "NameClaimType": "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/spn", // optional
+        "RoleClaimType": "roles" // optional
+      },
+      {
+        "ProviderName": "EasyAuthWithHeaderService",
+        "Enabled": true,
+        "NameClaimType": "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress", //optional
+        "RoleClaimType": "http://schemas.microsoft.com/ws/2008/06/identity/claims/role" //optional
```

Comment:
Is it by purpose, that the ClaimsPrincipal will have "new" role claims from type ClaimsType.Role after successful authentification? Even if I submit "RoleClaimType": "roles" here? 
Or to say it in another way: 
Is this setting only means the role claim type, which comes in by token(, cookie, header, ...) and has no effect on the output claims of the ClaimsPrincipal after authentification?

Reply:
The Output is always Role. See this [line](https://github.com/kirkone/KK.AspNetCore.EasyAuthAuthentication/blob/0ad56bfb858933276abeeed27784d522e0fef0b6/src/KK.AspNetCore.EasyAuthAuthentication/AuthenticationTicketBuilder.cs#L45). You can only define what is the input. So you can map groups for example to roles. But in your application it will always the role claim. Should we change this? Or is this Behavior okay? Maybe you can open an issue for that?

I will now add the documentation for that.

Reply:
see the commit https://github.com/kirkone/KK.AspNetCore.EasyAuthAuthentication/pull/31/commits/a786c7e1e58dfb384f2bad75ae8d99115713c3ee for the documentation changes